vsum/src/test/java/tools/vitruv/methodologisttemplate/vsum/ScenarioTwoTest.java

```diff
@@ -13,6 +13,10 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class ScenarioTwoTest extends AbstractVSUMExampleTest {
+    private static final int REQUIREMENT_1_LINE = 23;
```

Comment:
Hardcoded position of lines where we should check requirements.
On my machine, REQ001 appears at line 24, and REQ002 at line 23.
@TomWerm Is this the case on your machine, too?